src/main/charts/bitbucket/templates/statefulset.yaml

```diff
@@ -25,7 +25,7 @@ spec:
         {{- include "bitbucket.selectorLabels" . | nindent 8 }}
     spec:
       serviceAccountName: {{ include "bitbucket.serviceAccountName" . }}
-      terminationGracePeriodSeconds: 1
+      terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
```

Comment:
Should we add tests to ensure that the correct grace period (whats defined in `values.yaml`) is applied?

Reply:
This is done via the expected-output tests.